service-deployment/clusterconfig-example.yaml

```diff
@@ -160,6 +165,7 @@ machinelist:
     webportal: "true"
     prometheus: "true"
     grafana: "true"
+    pylon: "true"
```

Comment:
Can this proxy run on other node？

Reply:
Yes, as long as the node have a public IP address.

Reply:
Can this proxy run in the ha mode?

Reply:
It's stateless. So theoretically it can run in HA mode.